Services/Diagnostics/LoggingConfig.cs

```diff
@@ -12,39 +12,48 @@ public enum LogLevel
         Error = 40,
         Always = 1000
     }
-    
+
```

Comment:
nit: revert space

Reply:
I don't see a need for a change here. I removed white space but the effect is the same: there is a line between the end of the previous block and the start of the method on line 16.